cmd/kill/killer/kill_windows.go

```diff
@@ -0,0 +1,22 @@
+// +build windows
+
+package killer
+
+import (
+        "os"
+)
+
+func Kill(pids []int, safe bool) (map[int]error) {
+	var errm = make(map[int]error)
+	var err error
+
+	for _, pid := range pids {
+		proc := os.Process{Pid: pid}
```

Comment:
It's curious. In `_windows` Process.signal() exists, is called from `.Kill()` and calls `terminateProcess()`, but documentation says that on Windows [interrupt sending is not implemented](https://github.com/golang/go/blob/212d2f82e05018f1ebb5e40e2c328865201da356/src/os/exec.go#L118). Perhaps an outdated documentation?